plugin/pkg/scheduler/scheduler.go

```diff
@@ -185,7 +185,7 @@ func (sched *Scheduler) schedule(pod *v1.Pod) (string, error) {
 }
 
 // assume signals to the cache that a pod is already in the cache, so that binding can be asnychronous.
-func (sched *Scheduler) assume(pod *v1.Pod, host string) error {
+func (sched *Scheduler) assume(pod *v1.Pod, host string) (*v1.Pod, error) {
```

Comment:
sorry, I don't understand -- the reason this is returning the assumed pod is that `bind` expects to get the assumed pod as an argument (`bind(assumed *v1.Pod, b *v1.Binding)`). 

before the refactor assume and bind were part of the same function and the assumed pod got passed to `ForgetPod` and `FinishBinding` -- you can see the old code (which I believe was correct)  here: https://github.com/kubernetes/kubernetes/blob/80f26fa8a89ef5863cb19c71a620bb389d025166/plugin/pkg/scheduler/scheduler.go#L229-L234